server/src/diskdb.ts

```diff
@@ -59,9 +63,10 @@ export class Disk implements Repository {
   public async update(projectId: string, boardLink: string): Promise<string> {
     const project = this.db.get(projectId);
 
-    if (!project) {
+    if (!project || !isProject(project)) {
```

Comment:
Can you add the falsy check into `isProject`?

Reply:
Ill have to create a quick checklist function too as i did have a falsy check in there but that would invalidate what i needed for `is not quickchecklist`.